src/main/java/com/neonlab/product/service/OrderService.java

```diff
@@ -0,0 +1,49 @@
+package com.neonlab.product.service;
+import com.neonlab.common.dto.ApiOutput;
+import com.neonlab.common.expectations.InvalidInputException;
+import com.neonlab.common.expectations.ServerException;
+import com.neonlab.common.utilities.ObjectMapperUtils;
+import com.neonlab.product.controller.ProductController;
+import com.neonlab.product.dtos.BoughtProductDetailsDto;
+import com.neonlab.product.dtos.OrderDto;
+import com.neonlab.product.entities.Product;
+import com.neonlab.product.models.ProductDeleteReq;
+import com.neonlab.product.repository.OrderRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+
+
+@Service
+public class OrderService {
+    @Autowired
+    private OrderRepository orderRepository;
+
+    @Autowired
+    ProductService productService;
+    @Autowired
+    ProductController productController;
```

Comment:
I told to use `API` not controller. Please refactor the same to use `API`

*Note*:  We should never initialize an controller in any service.

cc- @sjha24